tests/test_kubernetes_tools.py

```diff
@@ -2073,6 +2067,17 @@ def test_parse_container_resources():
     ) == KubeContainerResources(2, 100, None)
 
 
+@mock.patch.object(kubernetes_tools, "sanitise_kubernetes_name", autospec=True)
```

Comment:
does this have to be mocked?

Reply:
Nope, but I want to assert that it was called.